operations/mimir/overrides-exporter.libsonnet

```diff
@@ -24,11 +25,20 @@
   local containerPort = $.core.v1.containerPort,
   overrides_exporter_port:: containerPort.newNamed(name='http-metrics', containerPort=$._config.server_http_port),
 
+  local ringConfig = {
```

Comment:
Our jsonnet support two ring backends: consul (default) and memberlist. Look for `querySchedulerRingClientConfig` and try to come up with something similar. Since in this case both lifecycler and ring client is only used by overrides exporter, you don't need two separate configs for the lifecycler and ring client, but a single `overridesExporterRingConfig` is enough. Make `overridesExporterRingConfig` not local, so that it's overrideable (like `querySchedulerRingClientConfig`).

Reply:
Then I would enable it by default with deployment mode is read-write, so we also see the difference in the jsonnet tests (run `make jsonnet-tests` to update the tests output).